.github/CODEOWNERS

```diff
@@ -988,6 +988,9 @@ sdk/ai/ai-inference-rest @glharper @dargilco @jhakulin
 # PRLabel: %Mgmt
 /sdk/chaos/arm-chaos/ @qiaozha @MaryGao
 
+# Cosmos API review files
+/sdk/cosmosdb/cosmos/review/*api.md @amanrao23 @aditishree1 @sajeetharan @topshot99
```

Comment:
Sorry I didn't make it clearer.  I suggested to move this after the following entry at Line 311

```
# Managment API review files
/sdk/**/review/arm-*.api.md @qiaozha @MaryGao @Azure/azure-sdk-for-js-core
```

Reply:
addressed @jeremymeng